### PR TITLE
search recipes added

### PR DESCRIPTION
**Pull request by AndriiPryslupskyi**

### CODE REVIEW

File: server/handler.go [hunk 1]
```diff
@@ -197,13 +210,43 @@ func getRecipes(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//for local work
```
Comment (anastasiia-skliar) [inline server/handler.go]: This is a bad approach to leave commented code.  You can move it to function and add flag to config (local or prod)

File: server/handler.go [hunk 2]
```diff
@@ -220,52 +263,53 @@ func updateAgent(w http.ResponseWriter, r *http.Request) {
...
 }
 
 func clientLogout(w http.ResponseWriter, r *http.Request) {
 	if err := closeSession(w, r); err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
+		sendErrorMsg(w, err, http.StatusInternalServerError)
 		return
 	}
-	w.WriteHeader(http.StatusOK)
+	sendErrorMsg(w, nil, http.StatusOK)
```
Comment (anastasiia-skliar) [inline server/handler.go:278]: sendErrorMsg(w, nil, http.StatusOK) - Really ? 

Either rename it to sendResponse, or create other to send OK

File: server/session.go [hunk 3]
```diff
@@ -45,16 +53,27 @@ func closeSession(w http.ResponseWriter, r *http.Request) error {
...
+}
+
+func getUserID(r *http.Request) (string, error) {
+	session, err := store.Get(r, sessionName)
+	if err != nil {
+		return "", err
 	}
-	return true, nil
+
+	return fmt.Sprint(session.Values[ID]), err
```
Comment (anastasiia-skliar) [inline server/session.go]: Are you using Sprint to convert ID to string? I suppose that there should be other way

File: server/handler.go (hunk 2 above)
Comment (AndriiPryslupskyi) [reply server/handler.go:278]: I know it and I`m going to change it after Oleksander's pull request merging.

**Review by ASeegull — APPROVED** [review]

**Pull request merged**